tests/integration/cots_test.py

```diff
@@ -1683,3 +1692,123 @@ def test_cots_add_first_stop_with_advance_pass_midnight():
         assert stop_times[-1].departure_status == "none"
         assert stop_times[-1].arrival == datetime(2012, 11, 21, 20, 46)
         assert stop_times[-1].departure == datetime(2012, 11, 21, 20, 46)
+
+
+def test_cots_purge_trip_and_rt(mock_rabbitmq):
+    """
+    Simple COTS post, then test the purge
+    """
+    cots_file = get_fixture_data("cots_train_96231_delayed.json")
+    res = api_post("/cots", data=cots_file)
+    assert res == "OK"
+
+    with app.app_context():
+        # Check there's really something before purge
+        assert len(RealTimeUpdate.query.all()) == 1
+
+        # Put an old (realistic) date to RealTimeUpdate object so that RTU purge affects it
+        rtu = RealTimeUpdate.query.all()[0]
```

Comment:
You might be able to do that instead.

```suggestion
        rtu = RealTimeUpdate.query.first()
```

Reply:
done in cbcc66e0cb0f9ccd89fb666ba8583a3a984c1efe